packages/zowe-explorer-ftp-extension/README.md

```diff
@@ -12,6 +12,22 @@ Enhancements:
 
 For more information, see [Changelog](https://github.com/zowe/vscode-extension-for-zowe/blob/master/packages/zowe-explorer-ftp-extension/CHANGELOG.md).
 
+## Prerequisites
+
+Host-side prerequisites:
+
+- Check to ensure that z/OS FTP service can be accessed remotely:
```

Comment:
I suggest arranging this pre-req in a form of a procedure. Here's the suggestion:

- Obtain remote access to z/OS FTP service. 

**Follow these steps:**

1. Connect to z/OS with the FTP client.
2. Run the following command in the FTP client:

```
rstat
```

3. Set the `JESINTERFACELevel` option to `2`
(Alternative) 3. Ensure that the `JESINTERFACELevel` option is set to `2`.

Otherwise, you can link from the `- Obtain remote access to z/OS FTP service.` prerequisite to a document that contains that abovementioned procedure.

Reply:
@IgorCATech I have added the procedure you suggested above. I think it is a good change. Thank you!